tests/Unit/Models/AiWorkflowClaimTest.php

```
@@ -0,0 +1,43 @@
+
+namespace StratFlow\Tests\Unit\Models;
+
+use PHPUnit\Framework\TestCase;
+use StratFlow\Core\Database;
+use StratFlow\Models\AiWorkflowClaim;
+
+class AiWorkflowClaimTest extends TestCase
+{
+    public function testAcquireReturnsTrueWhenInsertWinsClaim(): void
+    {
+        $db = $this->createMock(Database::class);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)